packages/account/src/Configs/terms-of-use-config.tsx

```diff
@@ -1,29 +1,27 @@
-import { getDefaultFields, isDesktop } from '@deriv/shared';
-
+import React from 'react';
```

Comment:
this file is also good with ts extension

Reply:
but we use here React types